docs/pages/enroll-resources/desktop-access/active-directory.mdx

```diff
@@ -628,6 +628,9 @@ To configure Teleport to protect access to Windows desktops:
    For a detailed description of the configuration fields, see
    [Desktop Configuration Reference](../../reference/agent-services/desktop-access-reference/configuration.mdx).
 
+   More information on the differences between the `addr` and `locate_server` options
+   is available in the [LDAP server selection](#ldap-server-selection) section.
```

Comment:
We generally want to avoid adding links within the numbered instructions so the user doesn't need to skip around the guide and experience unnecessary cognitive load. Is there a way to work these instructions into the numbered section? 

If this is difficult, no worries, but I would at least indicate in this paragraph what kind of information the user would expect to find in the LDAP server selection section so they can decide whether they need to navigate to that section.

Reply:
Thanks @ptgott, that looks good